examples/fluids/qfunctions/channel.h

```diff
@@ -153,27 +148,21 @@ CEED_QFUNCTION(Channel_Inflow)(void *ctx, CeedInt Q, const CeedScalar *const *in
 
     // -- Total Energy Density
     v[4][i] -= wdetJb * u_normal * (E + P);
-
-  }  // End Quadrature Point Loop
```

Comment:
Same comment about this comment

Reply:
Discussion [above](https://github.com/CEED/libCEED/pull/1539#discussion_r1548310072) applies to this as well.